_posts/2018-10-01-project-roles.md

```diff
@@ -0,0 +1,81 @@
+date:   2018-10-01 12:54:35 +0200
+category: about-us
+---
+
+An insight into your own company or team to evaluate your weak points is one of the greatest tools you can use to improve. We try to do it constantly using various ways (sprint retrospectives for example), but I’d like to think critical thinking about onesevles run in our blood. This is why we recently indentified a potential problem in the way we aproach to projects.
+
+It became apparent to us, that the roles we take in each project are unclear. There is a huge gray area, especially between team leaders, senior developers and project managers. We haven’t talked about it much so obviously there are some expectations that are not met on both sides.
+
+To challenge this we made a list of all project roles we feel are required, and we did this in two separate teams to combine different points of view. This list was strongly focused on team leader responsibilities first, but it became clear that it’s one sided, and we cannot just dump all this responsibilities on one person, and that it varies from project to project.
+
+## Project roles
+
+What we did instead is a general list of roles, and for now it goes as follows (in random order):
+
+### Helicopter view
+
+Basically knowing more that is required to do your tasks, both in and out of your department. This allows you to connect the dots and make decisions based on the project as a whole. You can’t coordinate tasks and teamwork without a helicopter view. For example, knowing the design progress so you can anticipate and plan accordingly. Or even more trivial example, knowing when one departament will finish their job, so another can start. Or to know when one person can help out another, because they’ve done something similar or their tasks are related somehow.
+
+### Responding to ad-hoc situations
+
+If project is on fire, you’re the first line of defence. You may (and probably need to) delegate it further, but you’re the rapid response team. This allows an easy way out for every team member if something goes wrong and an urgent fix is required, for example the project needs an additional developer, because the project is behind schedule. Without this role, the responsibility would be distributed among the whole team and as such it would be ineffective.
+
+### Reporting
+
+You can present project status to stakeholders and bring anyone up to date. This may relate only to certain fields of the project, such as development, design or other.
+
+### Documenting
+
+Updating knowledge on Confluence, so everyone is on the same page. Documenting has a broader meaning, but for the purpose of defining roles, let’s just stick to gathering all knowledge and arrangements in one place. This may be as trivial as meeting notes — someone has to do it. Without this role, there wouldn’t be a single source of truth about the project; all would be scattered between JIRA tasks, emails, bug trackers, slack, etc.
```

Comment:
"there wouldn’t be a single source of truth" - i tak ostatecznie będzie, wszystko bedzie w rownych miejscach. IMO to zdanie nie wzmacnia tezy, moze nawet oslabia

Reply:
przeredagowałem